tools/update_unicode_iso8859_tables.dart

```diff
@@ -0,0 +1,266 @@
+// Copyright (c) 2023, the Dart project authors.  Please see the AUTHORS file
```

Comment:
If we want this to be usable by other packages I think it needs to be in `bin/` instead of `tool/`